src/composables/authSignupFunction.js

```diff
@@ -0,0 +1,26 @@
+import { createUserWithEmailAndPassword, updateProfile } from "firebase/auth";
+import { appAuth } from "@/firebase/config";
+import { useState } from "react";
+
+function useAuthSignUp() {
+  const [addUser, setAddUser] = useState("");
+
+  const signup = ({ name, email, password }) => {
+    createUserWithEmailAndPassword(appAuth, email, password)
+      .then((userCredential) => {
+        updateProfile(userCredential.user, { displayName: name });
+      })
+      .then((userCredential) => {
```

Comment:
There's no need to handle this anymore. Since the useState is of no consequence to us.